### PR TITLE
Update interactivity utils docs

### PR DESCRIPTION
**Pull request by kullJul**
*Marked utils-interactivity-selections.md as deprecated.
*Updated utils-interactivity-selections mentions as deprecated.
*Updated utils-chart.md

### CODE REVIEW

**Comment by Court72** [comment]
@mberdugo

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

@MicrosoftDocs/public-repo-pr-review-team

**Review by mberdugo — APPROVED** [review]

File: powerbi-docs/developer/visuals/bookmarks-support.md [hunk 1]
```diff
@@ -46,20 +46,23 @@ The new selection state (or filter) is communicated through the `options.jsonFil
...
 ## Visuals with selection
 
+> [!NOTE]
+> InteractivityService has been deprecated.
+
 If your visual interacts with other visuals using [Selection](https://github.com/PowerBi-Projects/PowerBI-visuals/blob/master/Tutorial/Selection.md), you can add bookmark support in one of two ways:
 
-- Through [InteractivityService](#use-interactivityservice-to-restore-bookmark-selections) to manage selections, use the `applySelectionFromFilter`. This is the easier and preferred method.
-- Through [SelectionManager](#use-selectionmanager-to-restore-bookmark-selections), if your visual doesn't use **InteractivityService**.
+- Through [InteractivityService](#use-interactivityservice-to-restore-bookmark-selections) to manage selections, use the `applySelectionFromFilter`. This is deprecated method.
```
Comment (mberdugo) [inline powerbi-docs/developer/visuals/bookmarks-support.md:54]: Link needs to be changed to (#use-interactivityservice-to-restore-bookmark-selections---deprecated)

**Review by mberdugo — COMMENTED** [review]
There's one broken link because you changed the name of a header. Fix that, and I'll approve.
Thanks.

File: powerbi-docs/developer/visuals/bookmarks-support.md (hunk 1 above)
Comment (kullJul) [reply powerbi-docs/developer/visuals/bookmarks-support.md:54]: Seems like PR has already been merged, do I need to create the new one with this fix?

Comment (mberdugo) [reply powerbi-docs/developer/visuals/bookmarks-support.md:54]: @kullJul Yes. I accidentally merged it. If you can create a new PR with this fix that would be great. Thanks.

Comment (kullJul) [reply powerbi-docs/developer/visuals/bookmarks-support.md:54]: @mberdugo done -> https://github.com/MicrosoftDocs/powerbi-docs/pull/5124

**Pull request merged**